background_scripts/commands.js

```diff
@@ -551,6 +562,11 @@ const commandDescriptions = {
   moveTabLeft: ["Move tab to the left", { background: true }],
   moveTabRight: ["Move tab to the right", { background: true }],
 
+  setZoom: ["Set zoom to the value given after the command, like setZoom 2.25", { background: true }],
+  zoomIn: ["Increase zoom on current tab", { background: true }],
```

Comment:
I think we can drop "on current tab." Our zoom works the same as Chrome's and the Chrome UI doesn't make that distinction.

Although now that I think about it, in Chrome when changing the zoom of a site, Chrome will also update all open tabs which are showing that same site. I haven't yet tested this PR, but presumably our implementation doesn't do that. That would be a nice future change (outside of this PR) to make the behavior consistent.

Reply:
I agree, that wording was left from the original implementation, which tried to limit the zoom to the current tab. It is actually **incorrect** now, as I had *already changed the implementation to match the default behavior*. The behavior of all of these zoom commands should match the browser default, and it **does** zoom all pages of the current domain when you use `zi` or any other of these commands (I just tested in Chromium to make sure it is still working).